src/css/custom.css

```diff
@@ -22,6 +22,18 @@ body {
         rgba(0, 0, 0, 0.117647) 0px 1px 4px;
 }
 
+.search-bar {
+  margin-bottom: 24px;
+  width:100% !important;
+}
+
+/* images */
+.icon-search {
+  position: absolute;
```

Comment:
Haha. Sad times. For now, this works well in web. I wonder if I should use a `flex` layout for this............... let me see.